src/js/components/viewer/models/audio-media-object.jsx

```diff
@@ -123,67 +129,85 @@ var AudioMediaObject = React.createClass({
+                    return;
+                }
 
-        var self = this;
+                this.load(streamUrl);
 
-        soundCloud.streamUrl(mediaObject.url, function(err, streamUrl) {
-            if(err) {
-                throw err;
-                return;
-            }
+                // APEP TODO resolve audio volume bug then we can turn this back on
+                this.volume(volume);
 
-            var volume = mediaObject.volume;
-            if (isNaN(volume)) {
-                volume = 100;
-            }
-            volume = volume / 100;
-
-            self.state.player = new Audio5({
-                throw_errors: false,
-                format_time: false,
-                ready: function() {
-                    if(!self.state.playing) {
-                        return;
-                    }
+                self.unlockCuePointsForMediaObject(mediaObject);
 
-                    this.load(streamUrl);
+                this.play();
 
-                    this.volume(0);
+                //APEP tween volume TODO Turn back on with time to diagnose
+                var _ops = self.props.data.mediaObject._ops;
+                // self.state.playingAudioTween = self.tweenAudio(0, volume, _ops.transitionDuration, this);
+                // self.state.playingAudioTween.start();
 
-                    self.unlockCuePointsForMediaObject(mediaObject);
+                this.on('timeupdate', self.audioPlayerTimeUpdate);
 
-                    this.play();
+                this.on('error', function(err) {
+                    self.destoryAudioPlayer();
+                });
+            }
+        });
+    },
 
-                    var _ops = self.props.data.mediaObject._ops;
+    play: function() {
 
-                    //APEP tween volume
-                    self.state.playingAudioTween = self.tweenAudio(0, volume, _ops.transitionDuration, this);
-                    self.state.playingAudioTween.start();
+        var mediaObject = this.props.data.mediaObject._obj;
 
-                    this.on('timeupdate', self.audioPlayerTimeUpdate);
+        if(!this.state.playing) {
+            return;
+        }
 
-                    this.on('error', function(err) {
-                        self.destoryAudioPlayer();
-                    });
+        var self = this;
+
+        // APEP soundcloud audio - a simple search of the URL is OK.
+        if (mediaObject.url.indexOf("soundcloud.com") !== -1) {
+            soundCloud.streamUrl(mediaObject.url, function(err, streamUrl) {
+                if(err) {
+                    // APEP if we get an error from soundcloud, we will not be able to play the audio file. clean up immediately
+                    self.destoryAudioPlayer();
                 }
-            })
-        });
 
+                self.audio5Play(mediaObject, streamUrl);
+            });
+        } else {
+            // APEP TODO Must check if it is transcoded.  We will need the full scene object with appended DB values
```

Comment:
Currently purposeful, as we have not rolled out transcoded audio fully, we just attempt to play the source file yet.